### PR TITLE
PR re-issue: updated code content description for sdpv

### PR DESCRIPTION
**Pull request by cssandlin**
It looks like a few edits where track changed in lines 28, 40, and 49.  Updates were only made to the SDP-V content in line 177 and 184.  I'm not seeing any issues when testing locally per the lines above.

Thanks,
Chris

### CODE REVIEW

File: _data/code.yml [hunk 1]
```diff
@@ -174,14 +174,14 @@
...
-      description: Repository for the development of the initial SDP vocabulary service.
+      description: Repository for the development of the SDP Vocabulary Service.
       tags:
         - surveillance
         - javascript
       license: Apache License 2.0
     - title: SDP-CBR
       url: https://github.com/CDCgov/SDP-CBR
-      description: Repository for the development of the initial SDP Content-based Routing Service.
+      description: Repository for the development of the SDP Content-Based Routing Service.
```
Comment (leebrian) [inline _data/code.yml]: @cssandlin did you mean to put "retired" here?

**Comment by cssandlin** [comment]
I did!  Good catch, thanks!

**Comment by cssandlin** [comment]
added retired to CBR.

**Comment by leebrian** [comment]
Thanks sir. Sorry for the dupe work. I’ll try to merge better next time. This update will go into a new release in the next few days.

**Comment by cssandlin** [comment]
All good, thanks!

**Pull request merged**